flax/linen/fp8_ops.py

```diff
@@ -0,0 +1,207 @@
+from typing import Callable, Iterable, Union, Tuple
+from functools import partial
+
+from flax import core
+from flax.core import scope as flax_scope
+from flax.linen import initializers
+from flax.linen.module import Module
+from flax.linen import partitioning as nn_partitioning
+from jax import custom_vjp
+from jax import lax
+from jax import numpy as jnp
+from jax import random
+
+from fp8layers.jax_common.fp8 import fp8_dot_general
```

Comment:
This will cause module importing error and needs to be removed. Please clean up other unused imports.